staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -3062,7 +3167,7 @@ Rules:
   Host         Path  Backends
   ----         ----  --------
   foo.bar.com  
-               /foo   default-backend:80 (<error: endpoints "default-backend" not found>)
+               /foo   default-backend:80 (<error: services "default-backend" not found>)
```

Comment:
I had to update this as we call **list** on endpointslices rather than **get**, **list** returns zero endpointslices rather than error which **get** endpoint was returning previously.  
After listing endpoints we call **get** on service which throws the error.